modules/builds-create-imagestreamtag.adoc

```diff
@@ -3,29 +3,26 @@
 //* builds/running-entitled-builds.adoc
 
 [id="builds-create-imagestreamtag_{context}"]
-= Creating an image stream tag to the Red Hat Universal Base Image
+= Creating an imagestreamtag for the Red Hat Universal Base Image
 
-To use Red Hat subscriptions within a build, you should create an image stream to reference the universal base image (UBI).
+To use Red Hat subscriptions within a build, you create an imagestreamtag to reference the Universal Base Image (UBI).
 
-Builds that reference the UBI directly from registry.redhat.io require a pull secret.
+To make the UBI available *in every project* in the cluster, you add the imagestreamtag to the {product-title} namespace. Otherwise, to make it available *in a specific project*, you add the imagestreamtag to that project.
```

Comment:
the `openshift` namespace is always of that name, regardless of whether you are running OCP, OKD, whatever.

So instead of saying `{product-title} namespace` you should simple say `'openshift' namespace`